src/main/java/com/internet/shop/model/User.java

```diff
@@ -1,10 +1,13 @@
 package com.internet.shop.model;
 
+import java.util.List;
+
 public class User {
     private Long userId;
     private String name;
     private String login;
     private String password;
+    private List<Role> roles;
```

Comment:
let's make this field - Set
```suggestion
    private Set<Role> roles;
```